app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -147,6 +148,27 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPassword_pinView_hasContentDescription() {
```

Comment:
can we have a landscape version of this test? as we are setting content description in the land version of XML so it's different from the regular one.

Reply:
Added another test.

Reply:
All tests passed. Resolving this.